src/index.html

```diff
@@ -9,10 +9,60 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/modifiers.css"
+    />
```

Comment:
Ensure that the `modifiers.css` file exists in the `styles` directory. This file is crucial for the styling of the modifiers in your project.